pages/lists/[listId]/highlights.tsx

```diff
@@ -0,0 +1,267 @@
+      setContributor(searchTerm);
+    }
+  }
+
+  useEffect(() => {
+    onSearch(contributor);
+  }, [debouncedSearchTerm]);
+
+  return (
+    <ListPageLayout showRangeFilter={false} list={list} numberOfContributors={numberOfContributors} isOwner={isOwner}>
+      <div
+        ref={topRef}
+        className="container flex flex-col justify-start w-full gap-12 pt-4 mt-5 md:mt-0 md:items-start md:flex-row"
+      >
+        <div className={`sticky top-14 xl:flex hidden flex-none w-1/5`}>
+          <div className="w-full bg-white border shadow-sm rounded-lg px-5 py-6 flex flex-col gap-4">
+            <Search
+              value={contributor}
+              onChange={(value) => {
+                onSearch(value);
+              }}
+              placeholder="Search contributors"
+              className="!w-full"
+              name="helo"
+            />
+
+            {taggedRepos && taggedRepos.length > 0 ? (
+              <div className="flex flex-col gap-2">
+                <span className="text-sm text-light-slate-9">Tagged Repositories</span>
+                <SingleSelect
+                  placeholder="Select repository"
+                  options={taggedRepos.map((repo) => ({ label: repo.full_name, value: repo.full_name }))}
+                  onValueChange={(value) => {
+                    setQueryParams({ repo: value });
+                  }}
+                  value={repo ?? undefined}
+                  position="popper"
+                />
+              </div>
+            ) : null}
+
+            <div className="flex flex-col gap-2">
+              <span className="text-sm text-light-slate-9">Published</span>
+
+              <RadioGroup
+                onValueChange={(value) => {
+                  setQueryParams({ range: value });
+                }}
+                defaultValue={range as string}
+              >
+                {rangeFilterOptions.map(({ label, value }) => (
+                  <div key={value} className="flex items-center gap-2">
+                    <RadioGroupItem id={value} value={value}></RadioGroupItem>
+                    <label className="text-sm text-light-slate-12" htmlFor={value}>
+                      {label}
+                    </label>
+                  </div>
+                ))}
+              </RadioGroup>
+            </div>
+          </div>
+        </div>
+        <div className="w-full 2xl:max-w-[40rem] xl:max-w-[33rem] flex flex-col gap-10">
+          {isLoading ? (
+            <div className="flex flex-col gap-3">
+              <div className="flex gap-3">
+                <SkeletonWrapper radius={100} width={40} height={40} /> <SkeletonWrapper width={200} height={40} />
+              </div>
+              <SkeletonWrapper height={300} />
+            </div>
+          ) : null}
+
+          {!isLoading && data && data.length === 0 ? (
+            <DashContainer>
+              <p>You have no highlights for this list.</p>
```

Comment:
Instead of this message, consider adding the time range as there are highlights in some cases depending on the time range given.

![CleanShot 2023-12-12 at 16 44 38](https://github.com/open-sauced/app/assets/833231/0b7804f5-40ed-4530-80ce-c4f77d40432c)